addons/mail/static/tests/new/discuss/sidebar_tests.js

```diff
@@ -195,51 +175,44 @@ QUnit.test("sidebar: inbox with counter", async function (assert) {
 
 QUnit.test("default thread rendering", async function (assert) {
     const pyEnv = await startServer();
-    const mailChannelId1 = pyEnv["mail.channel"].create({ name: "" });
+    const channelId = pyEnv["mail.channel"].create({ name: "" });
     const { openDiscuss } = await start();
     await openDiscuss();
     assert.containsOnce(target, 'button[data-mailbox="inbox"]');
     assert.containsOnce(target, 'button[data-mailbox="starred"]');
     assert.containsOnce(target, 'button[data-mailbox="history"]');
-    assert.containsOnce(target, `.o-mail-category-item[data-channel-id="${mailChannelId1}"]`);
+    assert.containsOnce(target, `.o-mail-category-item[data-channel-id="${channelId}"]`);
     assert.hasClass($(target).find('button[data-mailbox="inbox"]'), "o-active");
-    assert.containsOnce(
-        target,
-        '.o-mail-discuss-content [data-empty-thread=""]',
-        "should have empty thread in inbox"
-    );
+    assert.containsOnce(target, '[data-empty-thread=""]');
     assert.strictEqual(
-        $(target).find('.o-mail-discuss-content [data-empty-thread=""]').text().trim(),
+        $(target).find('[data-empty-thread=""]').text().trim(),
         "Congratulations, your inbox is empty  New messages appear here."
     );
 
     await click('button[data-mailbox="starred"]');
     assert.hasClass($(target).find('button[data-mailbox="starred"]'), "o-active");
     assert.containsOnce(target, '.o-mail-discuss-content [data-empty-thread=""]');
     assert.strictEqual(
-        $(target).find('.o-mail-discuss-content [data-empty-thread=""]').text().trim(),
+        $(target).find('[data-empty-thread=""]').text().trim(),
         "No starred messages  You can mark any message as 'starred', and it shows up in this mailbox."
     );
 
     await click('button[data-mailbox="history"]');
     assert.hasClass($(target).find('button[data-mailbox="history"]'), "o-active");
-    assert.containsOnce(target, '.o-mail-discuss-content [data-empty-thread=""]');
+    assert.containsOnce(target, '[data-empty-thread=""]');
```

Comment:
assert not really clear, would be better to assert "There are no messages in this conversation." (or similar text in case of mailbox)

Reply:
Definitely not the best assertion. Improvement in follow-up PR